pages/index.tsx

```diff
@@ -11,16 +11,21 @@ import Mentee from '../components/landing/mentee';
 import SocialMedia from '../components/landing/socialMedia';
 import GetInvolved from '../components/landing/getInvolved';
 import Hero from '../components/landing/hero';
+import Banner from '../components/landing/banner/Banner';
 
 const Home: NextPage = () => (
 	<>
-		<SiteHead title="Underdog Devs" description="We are a group of software engineers supporting formerly incarcerated and disadvantaged aspiring developers" />
+		<SiteHead
+			title="Underdog Devs"
+			description="We are a group of software engineers supporting formerly incarcerated and disadvantaged aspiring developers"
+		/>
 		<div>
 			<Hero />
 			<Recidivism />
 			<Mentor />
 			<Mentee />
 			<SocialMedia />
+			<Banner />
```

Comment:
once we're done styling this up and get screenshots let's remove this from the main page before we merge this until we're ready for it to go live.